exonum-java-binding/core/rust/ejb-app/TUTORIAL.md

```diff
@@ -67,34 +67,34 @@ $ cargo run -- generate-template testnet/common.toml
 
 #### Generate Node Private and Public Configs
 
-- `--ejb-classpath` for a classpath of your service.
-- `--ejb-libpath` for a path to Java bindings native libraries.
-
 ```$sh
 $ cargo run -- generate-config testnet/common.toml testnet/pub.toml testnet/sec.toml \
-    --ejb-classpath $EJB_CLASSPATH \
-    --ejb-libpath $EJB_LIBPATH \
     --peer-address 127.0.0.1:5400
 ```
 
 #### Finalize Configuration
 
-There are two specific parameters here:
-- `--ejb-module-name` for fully qualified name of your `ServiceModule`.
-- `--ejb-port` for port that your service will use for communication.
-  Java Binding does not use Exonum Core ports directly.
+There is one specific parameter here:
+- `--ejb-artifact-uri` for the URI of Java service artifact.
```

Comment:
`uri` means that it supports some protocols i.e. ` ftp://ftp.xyz.com/dir/artifact`. Does it work in this way?

Reply:
Yeap, you're right.